src/index.scss

```diff
@@ -111,18 +111,14 @@
 	padding-right: 8px !important;
 }
 
-.block-editor-inserter__panel-header {
-	display: block;
-	border-bottom: 1px solid lightgray;
-	padding-bottom: 12px;
-}
-
```

Comment:
Removed as this subtle style override resulted caused oddities in the block appender's quick inserter styles.